EESSI-extend-2023.06-easybuild.eb

```diff
@@ -87,6 +87,16 @@ if (os.getenv("EESSI_CVMFS_INSTALL") ~= nil) then
   end
   eessi_cvmfs_install = true
   easybuild_installpath = os.getenv("EESSI_SOFTWARE_PATH")
+  eessi_accelerator_target = os.getenv("EESSI_ACCELERATOR_TARGET")
+  if (eessi_accelerator_target ~= nil) then
+    cuda_compute_capability = string.match(eessi_accelerator_target, "^nvidia/cc([0-9][0-9])$")
+    if (cuda_compute_capability ~= nil) then
+      easybuild_installpath = pathJoin(easybuild_installpath, 'accel', eessi_accelerator_target)
+      easybuild_cuda_compute_capabilities = cuda_compute_capability:sub(1, 1) ..  "." .. cuda_compute_capability:sub(2, 2)
+    else
+      LmodError("Incorrect value for $EESSI_ACCELERATOR_TARGET: " .. eessi_accelerator_target)
```

Comment:
If this is being set, I think I would disable `--robot` and print a message explaining why. Otherwise people run the risk of installing CPU software under the accelerator prefix unless they reload the module with `EESSI_ACCELERATOR_TARGET` unset again. They can always add back the option on the command line, but then they will consciously do so.

Reply:
Doing this via a hook is definitely a better idea. We would just need to figure out the condition to check for. I guess it would be CUDA _not_ in the dependencies but `/accel/` in the installation path.

Reply:
OK if I open an issue for that, e.g., to use an EB hook?